src/UI/WB.UI.Headquarters.Core/Models/ExportSettingsModel.cs

```diff
@@ -1,24 +1,27 @@
-﻿namespace WB.Core.SharedKernels.SurveyManagement.Web.Models
+﻿using WB.Core.BoundedContexts.Headquarters.DataExport.Security;
+
+namespace WB.Core.SharedKernels.SurveyManagement.Web.Models
 {
     public class ExportSettingsModel
     {
         public ExportSettingsModel(){}
 
-        public ExportSettingsModel(bool isEnabled, string password, bool isRetentionEnabled = false, 
-            int? retentionLimitInDays = null, 
-            int? retentionLimitQuantity = null)
+        public ExportSettingsModel(bool isEnabled, string password, ExportRetentionSettings exportRetentionSettings = null)

```

Comment:
listed of isEnabled and password pass exportSettings class

Reply:
there is no exportSettings class, but IExportSettings interface with methods 
bool EncryptionEnforced();
string GetPassword();